components/automate-cli/pkg/adminmgmt/adminmgmt_test.go

```diff
@@ -11,15 +11,11 @@ import (
 	"google.golang.org/grpc/status"
 
 	authz_constants "github.com/chef/automate/components/authz-service/constants"
-	authz_constants_v1 "github.com/chef/automate/components/authz-service/constants/v1"
 	"github.com/chef/automate/components/automate-cli/pkg/adminmgmt"
 	"github.com/chef/automate/components/automate-cli/pkg/client/mock"
-	teams_req "github.com/chef/automate/components/automate-gateway/api/auth/teams/request"
-	teams_resp "github.com/chef/automate/components/automate-gateway/api/auth/teams/response"
-	users_req "github.com/chef/automate/components/automate-gateway/api/auth/users/request"
-	users_resp "github.com/chef/automate/components/automate-gateway/api/auth/users/response"
 	authz_req "github.com/chef/automate/components/automate-gateway/api/authz/request"
 	authz_resp "github.com/chef/automate/components/automate-gateway/api/authz/response"
+	policies_common "github.com/chef/automate/components/automate-gateway/api/iam/v2/common"
```

Comment:
```suggestion
	iam_common "github.com/chef/automate/components/automate-gateway/api/iam/v2/common"
```
(and same with the req and resp imports below)